tools/wltests/test_series

```diff
@@ -786,12 +786,25 @@ match_sha1() {
 	COMMIT_SHA1=$1
 
 	c_info "Current kernel: "
-	CURRENT=$($ADB shell 'uname -a')
-	c_info "   $CURRENT"
+	c_info "   $($ADB shell 'uname -a')"
 
-	[[ $CURRENT = *$COMMIT_SHA1* ]] || return $EAGAIN
+	CURRENT=$($ADB shell 'uname -r | awk -F "-" "{print \$NF}"')
+	CURRENT=${CURRENT:1}
 
-	return $OK
+	[[ $COMMIT_SHA1 =~ $CURRENT* ]] && return $OK
```

Comment:
The above chunks can be moved in a separate patch, which corresponds to the change you already describe in the commit message when you say "It also inverts the substirng..." 

There are also some typos in the commit message, e.g: s/substirng/substring/

Reply:
Good.